docker-compose.yml

```diff
@@ -21,6 +21,8 @@ services:
       KAFKA_LISTENERS: 'PLAINTEXT://:9092,SASL_PLAINTEXT://:9093'
       KAFKA_ADVERTISED_LISTENERS: 'PLAINTEXT://localhost:9092,SASL_PLAINTEXT://localhost:9093'
       KAFKA_SASL_ENABLED_MECHANISMS: 'PLAIN,SCRAM-SHA-256,SCRAM-SHA-512'
+      KAFKA_AUTHORIZER_CLASS_NAME: 'kafka.security.auth.SimpleAclAuthorizer'
```

Comment:
Newer Kafka version (>2.4.0) are now using `kafka.security.auth.AclAuthorizer` something to keep in mind if Kafka is upgraded.